### PR TITLE
Script updates for pyside2-5.15.2.1-py3.10 on Linux aarch64 (ARM64)

### PR DESCRIPTION
**Pull request by spham-amzn**
**Build Command**
```
python3 ../3p-package-scripts/o3de_package_scripts/build_package.py --search_path . pyside2-5.15.2.1-py3.10-rev4-linux-aarch64 
```

**Build output**
```
Creating/Updating package: pyside2-5.15.2.1-py3.10-rev4-linux-aarch64
    Adding files to: "./packages/temp/temp_package_pyside2-5.15.2.1-py3.10-rev4-linux-aarch64"
    Created: ./packages/pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.tar.xz
    Created: ./packages/pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.tar.xz.SHA256SUMS
    Created: ./packages/pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.tar.xz.content.SHA256SUMS
    Created: ./packages/pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.PackageInfo.json
    - Validating package: pyside2-5.15.2.1-py3.10-rev4-linux-aarch64 in folder ./packages....
    - WARNING: "License" : "custom" in PackageInfo.json indicates a custom license.
          Most packages use one of the licences from SPDX.  Use 'custom' only as a last resort.
          Check https://spdx.org/licenses for the official list of license Identifiers.
          Consider searching online for terms inside the license file to see if its one of the ones from SPDX.
    Package Hash: 30bc9a653d452f5cb03903fb4469a7f774c2cf964cbc6eed77f37cd2bce10875 *pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.tar.xz
```

**Upload output**
```
Using bucket: ly-dev-3p-packages
Package: pyside2-5.15.2.1-py3.10-rev4-linux-aarch64 ...
    - Validating package: pyside2-5.15.2.1-py3.10-rev4-linux-aarch64 in folder ./packages....
    - WARNING: "License" : "custom" in PackageInfo.json indicates a custom license.
          Most packages use one of the licences from SPDX.  Use 'custom' only as a last resort.
          Check https://spdx.org/licenses for the official list of license Identifiers.
          Consider searching online for terms inside the license file to see if its one of the ones from SPDX.
    - Uploading pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.tar.xz...
    - Uploading pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.tar.xz.SHA256SUMS...
    - Uploading pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.tar.xz.content.SHA256SUMS...
    - Uploading pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.PackageInfo.json...
    - Uploaded package pyside2-5.15.2.1-py3.10-rev4-linux-aarch64.
```

### CODE REVIEW

File: package-system/pyside2/build_linux.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ then
 fi
 
 # Set the dependent clang compiler for the build script to use
-LLVM_INSTALL_DIR=$TEMP_FOLDER/libclang-release_130-based-linux-Ubuntu20.04-gcc9.3-x86_64/libclang
+LLVM_INSTALL_DIR=/usr/lib/llvm-6.0
```
Comment (cgalvan) [inline package-system/pyside2/build_linux.sh:30]: Can we remove the `additional_download_packages` for linux now if it's going to use `/usr/lib/llvm-6.0`?

File: package-system/pyside2/build_linux_aarch64.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    echo "with the following command:"
+    echo
+    echo "python3 build_package_image.py"
+    echo
+    exit 1
+fi
+
+# Set the dependent clang compiler for the build script to use
+#LLVM_INSTALL_DIR=$TEMP_FOLDER/libclang-release_130-based-linux-Ubuntu20.04-gcc9.3-x86_64/libclang
+#PATH=$LLVM_INSTALL_DIR/bin:$PATH
```
Comment (cgalvan) [inline package-system/pyside2/build_linux_aarch64.sh:31]: These commented lines can be removed

**Review by cgalvan — APPROVED** [review]

File: package-system/pyside2/build_linux.sh (hunk 1 above)
Comment (spham-amzn) [reply package-system/pyside2/build_linux.sh:30]: Good catch, this is a mistake and needs to be reverted

File: package-system/python/linux_aarch64/package [hunk 3]
```diff
@@ -0,0 +1 @@
+/home/ubuntu/github/3p-package-source/package-system/pyside2/temp/python-3.10.5-rev2-linux-aarch64/
```
Comment (lumberyard-employee-dm) [inline package-system/python/linux_aarch64/package:1]: What is this file for?

File: package-system/pyside2/build_linux_aarch64.sh [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    echo "with the following command:"
+    echo
+    echo "python3 build_package_image.py"
+    echo
+    exit 1
+fi
+
+# Set the dependent clang compiler for the build script to use
+export LLVM_INSTALL_DIR=/usr/lib/llvm-6.0
+export LLVM_CONFIG=/usr/bin/llvm-config-6.0
```
Comment (lumberyard-employee-dm) [inline package-system/pyside2/build_linux_aarch64.sh:31]: llvm 6?
Shouldn't this be llvm 12

File: package-system/python/linux_aarch64/package (hunk 3 above)
Comment (spham-amzn) [reply package-system/python/linux_aarch64/package:1]: This file shouldn't be here

**Comment by spham-amzn** [comment]
This PR is not ready yet

**Pull request closed without merging**